src/System.Private.CoreLib/src/System/Reflection/Emit/DynamicILGenerator.cs

```diff
@@ -732,10 +733,10 @@ private static int CalculateNumberOfExceptions(__ExceptionInfo[] excp)
 
                 if ((header & 0x40) != 0) // Fat
                 {
-                    byte[] size = new byte[4];
+                    Span<byte> size = stackalloc byte[4];
```

Comment:
You should not need to copy the bytes from m_exceptionHeader to `Span<byte>`. `ReadInt32LittleEndian` can be called directly on a slice made out of `m_exceptionHeader`.

Reply:
You are right, oversight on my part, will adjust!

Reply:
Adjusted in https://github.com/dotnet/coreclr/pull/23833/commits/069fe276f583db34a22585742c0845444b21488d